CHANGES.md

```diff
@@ -1,3 +1,29 @@
+# lang_tester 0.5.0 (xxxx-xx-xx)
+
+* The `test_extract` function signature has changed from:
+  ```
+  Fn(&str) -> Option<String> + Send + Sync,
+  ```
+  to:
+  ```
+  Fn(&Path) -> String + Send + Sync,
+  ```
+
+  In other words, users now have to a) read the contents of a path themselves
+  (but it doesn't necessarily have to be the path passed to the function!) b)
+  return a `String` rather than an `Option<String>`.
```

Comment:
Can we try rephrasing this? It took me a couple of re-reads to parse correctly as I read the "a)" and "b)" as mutually exclusive options. Even just switching to _"users now have to do two things: ...."_ would help.

Reply:
Is https://github.com/softdevteam/lang_tester/pull/77/commits/937290345004d94d4b080aa0e6f66002b397ac31 enough or is more needed?

Reply:
Yes that's much better. Can you change it in the commit message too when you squash.

Reply:
Will do!